src/libraries/LogCompression.sol

```diff
@@ -39,18 +39,22 @@ library LogCompression {
      * value required.
      */
     function toLowResLog(uint256 value) internal pure returns (int256) {
+    unchecked {
         int256 ln = LogExpMath.ln(int256(value));
 
         // Rounding division for signed numerator
         int256 lnWithError = (ln > 0 ? ln + _HALF_LOG_COMPRESSION_FACTOR : ln - _HALF_LOG_COMPRESSION_FACTOR);
         return lnWithError / _LOG_COMPRESSION_FACTOR;
     }
+    }
 
```

Comment:
gas optimization or causing test failures?

Reply:
optimization, as written above in my PR description... no failure....